stock_picking_invoice_link/models/stock_move.py

```diff
@@ -10,16 +10,6 @@
 class StockMove(models.Model):
     _inherit = "stock.move"
 
-    # Provide this field for backwards compatibility
-    invoice_line_ids = fields.Many2many(
```

Comment:
Shouldn't it be a migration script to avoid old data loss?

Reply:
Same data is on field invoice_line_id, is redundant field